rpcs3/Emu/Cell/SPUCommonRecompiler.cpp

```diff
@@ -543,9 +543,26 @@ extern void utilize_spu_data_segment(u32 vaddr, const void* ls_data_vaddr, u32 s
 		return;
 	}
 
-	for (u32 addr : obj.funcs)
+	if (spu_log.notice && !obj.funcs.empty())
 	{
-		spu_log.notice("Found SPU function at: 0x%05x", addr);
+		std::string to_log;
+		usz i = 0;
+
+		for (u32 addr : obj.funcs)
+		{
+			if (i++ % 8 == 0)
+			{
+				to_log += '\n';
+			}
+			else
+			{
+				to_log += ", ";
+			}
+
+			fmt::append(to_log, "0x%08x", addr);
+		}
+
+		spu_log.notice("Found SPU functions at:%s", to_log);
```

Comment:
You can remove the message below if you just add the count here.

Reply:
This was supposed to be a success log message, changed it.
That's why it's two messages, so at least one tells the count in more visible manner.